proposals/exception-handling/Exceptions.md

```diff
@@ -450,14 +450,16 @@ tag, the associated data fields cannot be read.
 
 The `Exception` constructor can take an optional `ExceptionOptions` argument,
 which can optionally contain `traceStack` entry. When `traceStack` is `true`,
-web VMs can attach a stack trace string to `Exception.stack` field, as in
-JavaScript's `Error` class. While `Exception` is not a subclass of JavaScript's
-`Error` and it can be used to represent normal control flow constructs,
-`traceStack` field can be set when we use it to represent errors. The format of
-stack trace strings conform to the [WebAssembly stack trace
-conventions](https://webassembly.github.io/spec/web-api/index.html#conventions).
-When `ExceptionOption` is not provided or it does not contain `traceStack`
-entry, `traceStack` is considered `false` by default.
+JavaScript VMs are permitted to attach a stack trace string to `Exception.stack`
+field, as in JavaScript's `Error` class. `traceStack` serves as a request to the
+WebAssembly engine to attach a stack trace; it is not necessary to honour if `true`,
```

Comment:
While it is a request because there can be engines that don't have the stack trace capability, I'm not sure if the current text is unclear, or if I prefer to explicitly say "it is not necessary to honour", which is something we certainly don't encourage.

I'm actually not very sure why you find the current text not clear.

Reply:
Hmm, I see, I think that makes sense. By the way can you wrap the text to 80 cols? It's not a rule kept universally in wasm spec repo, but this file conforms to it, so I think it's better to be consistent.